### ISSUE TITLE
Saplings autoplanting mods doesn't work

### ISSUE THREAD

**Issue by Prooty** [GregTech6/gregtech6#111]
Hello, tried two mods that autoplant dropped tree saplings after certain amount of time, but saplings just disappearing on planting event. Same result with single gt6 and default configs.

https://github.com/Lemonszz/AutoSapling
https://github.com/MikhailTapio/QuickPlant

[fml-client-latest.log](https://github.com/GregTech6/gregtech6/files/12270502/fml-client-latest.log)
[latest.log](https://github.com/GregTech6/gregtech6/files/12270503/latest.log)

**Comment by GregoriusT** [comment]
Yeah those Mods check for normal Saplings via inheritance, but my Saplings dont inherit the vanilla Saplings Code resulting in this Issue.

**Comment by Prooty** [comment]
Woopsie, closed by accident. Was trying to write that autoplanting would be cool as feature in gt
